nano/node/bootstrap.cpp

```diff
@@ -14,6 +14,11 @@ constexpr double bootstrap_minimum_termination_time_sec = 30.0;
 constexpr unsigned bootstrap_max_new_connections = 10;
 constexpr unsigned bulk_push_cost_limit = 200;
 
+size_t constexpr nano::frontier_req::size;
+size_t constexpr nano::bulk_pull_blocks::size;
+size_t constexpr nano::bulk_pull_account::size;
+size_t constexpr nano::frontier_req_client::size_frontier;
```

Comment:
Are these forward declarations needed if we `#include <node/common.hpp>` which includes the definition ?

Reply:
You need a definition since we odr-use these (you'll get a linker error if not) We do this several places. The alternative is an inline constexpr function (or C++17 inline vars), but I think this is fine for now.

Reply:
After the most recent change, all that should be required here is `size_frontier`, correct ?

Reply:
Yeah, that's true. I'll fix that.